src/styles/blocks/catalog.scss

```
@@ -18,8 +20,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $transform-timing;
+
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    &:hover {
+      transform: scale(120%);
+      color: #34568b;
```

don't remember to create variables for colors
```suggestion
      color: $c-someColor;
```